src/bin/path_filestat.rs

```diff
@@ -106,16 +106,16 @@ unsafe fn test_path_filestat(dir_fd: wasi_unstable::Fd) {
     assert_eq!(stat.st_mtim, new_mtim, "mtim should change");
     assert_eq!(stat.st_atim, old_atim, "atim should not change");
 
-    assert!(
+    assert_eq!(
         wasi_path_filestat_set_times(
             dir_fd,
             0,
             filename,
             new_mtim,
             new_mtim,
             wasi_unstable::FILESTAT_SET_MTIM | wasi_unstable::FILESTAT_SET_MTIM_NOW,
-        )
-        .is_ok(),
+        ),
+        Err(wasi_unstable::EINVAL),
         "ATIM & ATIM_NOW can't both be set"
```

Comment:
The test is testing *m*tim, but the error message says *A*TIM. Also, should we test that you can't set `ATIM` and `ATIM_NOW` too?

Reply:
Good idea, thanks!